patches/net-url_request-url_request_context_builder.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/net/url_request/url_request_context_builder.cc b/net/url_request/url_request_context_builder.cc
+index 800515a58cfa8546288d961fc9d14ae4676fcb5f..c75eec623aa0aff28ed8fd0607af082bc58de768 100644
+--- a/net/url_request/url_request_context_builder.cc
++++ b/net/url_request/url_request_context_builder.cc
+@@ -621,6 +621,8 @@ std::unique_ptr<URLRequestContext> URLRequestContextBuilder::Build() {
+   }
+ #endif  // !BUILDFLAG(DISABLE_FTP_SUPPORT)
+ 
++  job_factory->SetProtocolHandler("magnet", std::make_unique<MagnetProtocolHandler>());
```

Comment:
this should not be added here. See ProfileIOData::AddProtocolHandlersToBuilder and ProfileIOData::InstallProtocolHandlers